example/mycars.jv

```diff
@@ -0,0 +1,83 @@
+// Define a new blocktype named CSVExtractor outside of the pipeline
```

Comment:
Nice example of composite blocktypes. I'd be happy if we can use this example later, but it needs some polishing first (and renaming)